scripts/headshot_scraper.js

```diff
@@ -20,14 +20,14 @@ var download = function(uri, filename, player, items, player_type, iteration, le
     if (content_type == 'image/jpeg'){
         var team_id = items.team_id;
         //This is really dirty. Mongo doesnt allow iterative variables in the update so we trick it by casting the object before asking mongo
-        avatar_url[player_type + "."+ iteration +".avatar_url"] = '../public/images/headshots/'+league+'/'+player.full_name.replace(/\s+/g, '-').toLowerCase()+'.jpg';
+        avatar_url[player_type + "."+ iteration +".avatar_url"] = '../images/headshots/'+league+'/'+player.full_name.replace(/\s+/g, '-').toLowerCase()+'.jpg';
     	request(uri).pipe(fs.createWriteStream(filename)).on('close', function(){
         });
         
     }
     else{
         var team_id = items.team_id;
-        avatar_url[player_type+"."+ iteration +".avatar_url"] = '../public/images/headshots/untitled.jpg';
```

Comment:
because, this is the url that will be referenced FROM the actual quiz page. Not the scripts directory. The other url's point of reference is the scripts dir.